openwisp_notifications/templatetags/notification_tags.py

```diff
@@ -32,4 +32,23 @@ def unread_notifications(context):
     return output
 
 
+def notification_widget(context):
+    return format_html(
+        '''
+        <div class="notification-dropdown hide">
+            <div class="filters">
+                <span class="btn" id="mark-all-read">Mark all read</span>
```

Comment:
disable onClick event when these buttons are disabled. Currently theses buttons send API request onClick even if they are disabled.
Also either use `<button>` tag or add role and tabindex attributes to them